nemo_skills/prompt/config/eval/bigcodebench/codegen_reasoning.yaml

```diff
@@ -0,0 +1,8 @@
+system: ""
```

Comment:
Do we really need two versions of these prompts? Why not keep just the reasoning one if it's probably always better to use it? Same question for lcb by the way

Reply:
Let's keep them separate for now.

Reply:
Are you actually using both? Or you're only using the reasoning one? If we use only 1 of them 99% of the time, I'd just keep that one to avoid growing the number of configs we have in main. Users can always create a custom prompt and provide it if they need.

Another question is that shouldn't the reasoning one be default in lcb as well?